addons/test_mail/tests/test_mail_flow.py

```diff
@@ -279,24 +279,28 @@ def test_lead_mailgateway(self):
                 },
             ],
         )
-        # expected Msg['To'] : actual recipient, same as SMTP To
+        # expected Msg['To'] : Reply-All behavior: actual recipient, then
+        # all "not internal partners" and catchall (to receive answers)
         for partner in responsible_answer.notified_partner_ids:
+            exp_msg_to_partners = partner | external_partners
+            exp_msg_to = exp_msg_to_partners.mapped('email_formatted')
             with self.subTest(name=partner.name):
                 self.assertSMTPEmailsSent(
                     mail_server=self.mail_server_notification,
-                    msg_from=formataddr(
-                        (self.partner_employee.name, f'{self.default_from}@{self.alias_domain}')
-                    ),
+                    # msg_from=formataddr(
```

Comment:
why that change ?

Reply:
At some point I had issues with some encoding but it does not appear anymore. I think I was comparing string and stringified lists, and "repr" made things crash. Solved anyway, probably an issue in test tools at that time.